docs/src/apireference.md

```diff
@@ -359,3 +359,66 @@ Bridges.SOCtoPSDBridge
 Bridges.RSOCtoPSDBridge
 ```
 For each bridge defined in this package, a corresponding bridge optimizer is available with the same name without the "Bridge" suffix, e.g., `SplitInterval` is an `SingleBridgeOptimizer` for the `SplitIntervalBridge`.
+
+## Copy utilities
+
+The following utilities can be used to implement [`copy_to`](@ref). See
+[Implementing copy](@ref) for more details.
+
+```@docs
+Utilities.automatic_copy_to
+Utilities.default_copy_to
+Utilities.supports_incremental_copy
+```
+
+### Allocate-Load API
+
+The Allocate-Load API allows solvers that do not support loading the problem
+incrementally to implement [`copy_to`](@ref) in a way that still allows
+transformations to be applied in the copy between the cache and the
+model if the transformations are implemented as MOI layers implementing the
+Allocate-Load API, see [Implementing copy](@ref) for more details.
+
+Loading a model using the Allocate-Load interface consists of two passes
+through the model data:
+1) the allocate pass where the model typically records the necessary information
+   about the constraints and attributes such as their number and size.
+   This information may be used by the solver to allocate datastructures of
+   appriate size.
+2) the load pass where the model typically loads the constraints and attributes
+   data to the model.
+
+The description above only gives a suggestion of what to achieve in each pass.
+In fact the exact same constraints and attributes are passes in each pass so
```

Comment:
"In fact, the same constraint and attribute data is provided to each pass, so an implementation of ..."